cost/google/idle_persistent_disk_recommendations/google_idle_persistent_disk_recommendations.pt

```diff
@@ -463,13 +463,13 @@ policy "policy_recommendations" do
     export "instances" do
       resource_level true
       field "accountID" do
-        label "Account ID"
-      end
-      field "accountNumber" do
-        label "Account Number"
+        label "Project ID"
       end
       field "accountName" do
-        label "Account Name"
+        label "Project Name"
+      end
+      field "projectNumber" do
```

Comment:
cannot find a field called `projectNumber`, can only see `accountNumber` referenced previously in the template